lib/src/connect/meeting/main_websocket/chat/chat.dart

```diff
@@ -347,9 +347,12 @@ class ChatModule extends Module {
     _activeChatGroups.add(group.id);
     _chatGroupController.add(ChatGroupEvent(group, true));
 
+    List<ChatGroup> chatGroupsTmp = [];
+    chatGroupsTmp.add(group);
+
     // Subscribe to messages of the chat group
     List<dynamic> params = chatID != defaultChatID
-        ? [_chatGroups.map((e) => "${e.id}").toList(growable: false)]
+        ? [chatGroupsTmp.map((e) => "${e.id}").toList(growable: false)]
```

Comment:
That is not like the web app behaves. The web app sends a list of all chat groups on each new private chat. Have you tested, that the other private chats still work after you do this? If that is the case I am fine with the changes 😄 

Another small thing: Why not just do something like `List()..add(group)` instead of using a new variable `chatGroupsTmp`. Like so:

```dart
    // Subscribe to messages of the chat group
    List<dynamic> params = chatID != defaultChatID
        ? [List()..add(group).map((e) => "${e.id}").toList(growable: false)]
        : [];
```

Or in this case it's probably better to just do:

```dart
    // Subscribe to messages of the chat group
    List<dynamic> params = chatID != defaultChatID
        ? [["${group.id}")]]
        : [];
```

Reply:
The web app sends a list of all chat groups, yes. But only the private chat groups. Not the MAIN_PUBLIC_GROUP_CHAT, which we included in our list. This caused the "double-sub", which is why we received messages from other meetings.

Tested, that it is enough to only send the id of the new chat group. Multiple chat groups still work.

Reply:
The web app sends a list of all chat groups, yes. But only the private chat groups. Not the MAIN_PUBLIC_GROUP_CHAT, which we included in our list. This caused the "double-sub", which is why we received messages from other meetings.

Tested, that it is enough to only send the id of the new chat group. Multiple chat groups still work.